librz/core/cmd_open.c

```diff
@@ -327,10 +326,6 @@ static void cmd_open_bin(RzCore *core, const char *input) {
 		break;
 	}
 	case 'r': // "obr"
-		rz_core_bin_rebase(core, rz_num_math(core->num, input + 3));
```

Comment:
The function `rz_core_bin_rebase` was used only here, I suggest to remove it completely. Moreover, I suggest to move `core_bin_reload()` function to the `librz/core/cbin.c` and making it an API - `RZ_API bool rz_core_bin_reload()`.

Reply:
Better to the `rz_core_file_reload()` and into `librz/core/cfile.c`, since it operates on the file, not `RzBin`.